docker-compose/_posts/2017-05-05-adding-a-database-for-compose.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Adding a Database for Compose
+short: Adding a Database
+category: docker-compose
+order: 2
+---
+
+Learn how to add a database to your Compose based environment.
+
+> If you’re **not using Compose**, follow the [databases for Dockerfiles guide](https://runnable.com/docs/databases-datastores/how-to-setup-your-postgres-database-template).
+
+---
+
+Start by creating a Dockerfile to build your service and adding it to your repository:
+
+<div class="grid-block align-center pre-header">
```

Comment:
![image](https://cloud.githubusercontent.com/assets/7440805/25865346/70582e4e-34a7-11e7-880d-32c73b297384.png)
It shows both code blocks in this format with JavaScript off. Is that what you want?